commcare_connect/templates/export_modal.html

```diff
@@ -0,0 +1,27 @@
+{% load crispy_forms_tags %}
+
+{% block export_modal %}
+  <!-- Export Template -->
+  <div class="modal fade" id="{{modal_id}}" data-bs-backdrop="static" data-bs-keyboard="false" tabindex="-1" aria-hidden="true">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h1 class="modal-title fs-5">{{modal_title}}</h1>
+          <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+        </div>
+        <form action="{{export_url}}" method="post"
+              enctype="multipart/form-data">
+          <div class="modal-body">
+            <div class="mb-3">
+              {% crispy user_status_export_form %}
```

Comment:
nit: we can probably rename this since it is used in a lot of exports now

Reply:
Thanks, i renamed this and refactored the code to use the same export form. It was already using the same form but the code was duplicated.